frontend/src/ui/components/Header/Header.tsx

```diff
@@ -1,25 +1,53 @@
 import * as React from 'react';
-import { Container, Nav, Navbar } from 'react-bootstrap';
-import { CONTACT_US_LINK, FAQ_PAGE_LINK, ROUTES } from 'global/constants';
+import { Container, Nav, Navbar, NavDropdown } from 'react-bootstrap';
+import { FAQ_PAGE_LINK, ROUTES } from 'global/constants';
 import { NavLink, Route } from 'react-router-dom';
+import { InstancesModel } from 'console/models';
+import { RootState } from 'global/state';
+import { connect } from 'react-redux';
 import logo from 'assets/icons.svg';
+import { DeploymentIndicator } from '../CustomStatusPill';
 import './styles.scss';
 
-export const Header: React.FC = () => {
+interface StateProps extends InstancesModel {}
+
+interface Props extends StateProps {
+  children: React.ReactNode;
+}
+
+export const HeaderComponent: React.FC<Props> = (props: Props) => {
   // Only show login link when on registration page and only show registration
   // link when on login page
+
+  let lmsLink = null;
+  let deploymentPill = null;
+
+  if (!props.loading && props.activeInstance.data !== null) {
+    // if active instance data is available, get lms link
+    lmsLink = props.activeInstance.data.lmsUrl;
```

Comment:
Nit: could we keep the naming, i.e. `lmsUrl`?

Reply:
Done